apps/src/aichat/views/UserChatMessageEditor.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '@cdo/apps/templates/Button';
+import moduleStyles from './userChatMessageEditor.module.scss';
+
+const UserChatMessageEditor: React.FunctionComponent = () => {
+  return (
+    <div className={moduleStyles.UserChatMessageEditor}>
+      <textarea
+        className={moduleStyles.textArea}
+        placeholder="Add a chat message..."
```

Comment:
localization for this too?

Reply:
Oh yes! I'll add the localized string in my follow-up PR. Thank you so much for all your feedback and help!